### PR TITLE
Carthage support

### PR DESCRIPTION
**Pull request by uny**
I would like to use this via Carthage.

### CODE REVIEW

File: Reachability/Reachability.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ POSSIBILITY OF SUCH DAMAGE.
 import SystemConfiguration
 import Foundation
 
-enum ReachabilityError: ErrorType {
+public enum ReachabilityError: ErrorType {
```
Comment (uny) [inline Reachability/Reachability.swift]: It should be public.

**Comment by ashleymills** [comment]
OK, looks like Carthage is a thing! I'll take a look at this PR and #56 and see what we can do

Cheers
Ash

**Comment by ashleymills** [comment]
I've merged this pull request into the develop branch - https://github.com/ashleymills/Reachability.swift/tree/develop

Please test and raise any issues in this comment thread.

Cheers
Ash

**Comment by uny** [comment]
Thank you.
It works properly.

**Comment by filipejtdias** [comment]
Great job!

**Comment by ashleymills** [comment]
It would be great if someone could update the the readme with Carthage installation instructions?

Cheers
Ash

**Comment by uny** [comment]
Yes, I will tomorrow.

**Pull request closed without merging**